project.clj

```diff
@@ -1,7 +1,7 @@
 (def schema-tools-version "0.9.1")
 (def schema-version "1.1.7")
 
-(defproject threatgrid/ctim "1.1.3-SNAPSHOT"
+(defproject threatgrid/ctim "1.1.5-SNAPSHOT"
```

Comment:
Why do you bump from `1.1.3-SNAPSHOT` to `1.1.5-SNAPSHOT`? A mistake in the release process with `1.1.3`?

Reply:
exactly :s
I make first commit 1.1.3 that I deployed to clojars
then commit 1.1.4-SNAPSHOT
At this point I realized that I forgot to rebase, thus I did not have the changes that I was really supposed to release...
So I rebase made a commit 1.1.4 from which I made a release then bumped to 1.1.5-SNAPSHOT.
I hope that my brain is less twisted than my ankle :-)